### ISSUE TITLE
Question: Is it working for snapshots? Like 20w14?

### ISSUE THREAD

**Issue by ThatBigDerp** [Hexeption/MCP-Reborn#223]
Is it working for snapshots like 20w14inf? Or its only for full versions?

**Comment by LionTion** [comment]
I don't have enough experience with decompilations or deobfuscating things

A tool that atleast tries to use the mappings provided by mojang to decompile any snapshot would be cool though :)

**Comment by vinc3nz** [comment]
Yes it (theoratically) works for specific snapshots. To run on a specific snapshot head over to[ MinecraftForge/MCPConfig](https://github.com/MinecraftForge/MCP-Config) and search for the required snapshot version. Then change the mcp-version in your build.gradle to the version of MCP-Config. Just checked: It seems that Forge does not publish thoose Snapshots to their maven. However with some work you should get it to work.

**Comment by gabriel-fondato** [comment]
> Yes it (theoratically) works for specific snapshots. To run on a specific snapshot head over to[ MinecraftForge/MCPConfig](https://github.com/MinecraftForge/MCP-Config) and search for the required snapshot version. Then change the mcp-version in your build.gradle to the version of MCP-Config. Just checked: It seems that Forge does not publish thoose Snapshots to their maven. However with some work you should get it to work.

how exactly would you do that? on build.gradle i saw a maven repo and you said that forge dont publish those snapshots on their maven and i cant check it out because it requires an login and on the mcpconfig repo i got to version/snapshot/april and then the snapshot folder with the files that i want but what exactly do i need to copy to the build.gradle? do i need to change the maven repo? and there are some variables there like minecraft_version, mcp_version, mappings_channel, mappings_version and spi_version. where do i get the stuff to put there? i didnt find those veriables in any file in the snapshot/20w14infinite folder on the MinecraftForge/MCPConfig
/
[MCPConfig](https://github.com/MinecraftForge/MCPConfig)
